### ISSUE TITLE
Masurca error

### ISSUE THREAD

**Issue by sekhwal** [alekseyzimin/masurca#273]
Hi, 
I am using masurca for making hybrid assembly. However, it shows following error. Please let me know how to figure out the issue.

Run command:

/bin/masurca -t 32 -i /st_R1.fastq.gz,/st_R2.fastq.gz -r /nanopore_filter.fastq.gz

------error-------
[Fri Jan 14 22:08:48 EST 2022] Failed to pre-correct long reads, please check your input files /mnt/d/DMV_virus/Galaxy_Orf_nanopore_filter.fastq.gz and pe.cor.fa and the other parameters
[Fri Jan 14 22:08:48 EST 2022] Failed to pre-correct /mnt/d/DMV_virus/Galaxy_Orf_nanopore_filter.fastq.gz file, please check your data!
[Fri Jan 14 22:08:48 EST 2022] mega-reads exited before assembly

**Comment by chris-dumigan** [comment]
Did you find any solution to this? I have identical commands and data types and am getting same error on masurca 4.0.9

**Comment by bndaniel** [comment]
Most likely missing the tool numactl

**Comment by vanessa63104** [comment]
Bonjour,
Avez vous trouvez une solution, j'ai le même problème et j'ai installé l'outil numactl

**Comment by chris-dumigan** [comment]
To confirm the above suggestions, I believe I fixed this error with masurca in a conda environment on a centos server by installing numactl into the base environment with: 

yum install numactl

**Comment by salvatierra8** [comment]
I'm having a similar error, numactl is installed, but the error keeps being the same

**Comment by vanessa63104** [comment]
Avez vous la dernière version ? je n'ai pas encore trouvé de solution pour ce problème

**Comment by salvatierra8** [comment]
Yes. i have the latest version installed, but although masurca runs a little more steps than 4.0.9, it still gives the same error. I don't know how this is working, a companion of mine had the same error in previous version too, now she managed to run her hybrid assembly... i'm still figuring out what's going on

**Comment by vanessa63104** [comment]
Bonjour salvatierra8,
Avez vous réussi à faire fonctionner masurca?

**Comment by salvatierra8** [comment]
Hello, no I hadn't had any luck. I went onto using another hybrid assembler without much problems, but still it would be cool to use Masurca and compare the results, as of now i have a lot of misassemblies.
